client/i18n/locales/english/translations.json

```
@@ -536,7 +543,8 @@
     "theme-updated": "Your theme has been updated!",
     "username-used": "Username is already associated with this account",
     "username-taken": "Username is already associated with a different account",
-    "username-updated": "We have updated your username to {{username}}",
+    "username-updated": "Your username is updated to {{username}}",
+    "privacy-updated": "Your privacy setting has been updated",
```

```suggestion
    "username-updated": "We have updated your username to {{username}}",
    "privacy-updated": "We have updated your privacy settings",
```
This should fix the failing `username-change` tests.